projects/1.20/assets/skays-overworld-netherite-ore/overworld_netherite_ore/lang/zh_cn.json

```
@@ -0,0 +1,25 @@
+{
+  "advancements.master_of_ancient_shards.descr": "获得一组大型远古残骸碎片.",
+  "advancements.master_of_ancient_shards.title": "远古残骸碎片大师",
+  "advancements.overworld_netherite.descr": "感谢您下载 Overworld Netherite! <3",
```

```suggestion
  "advancements.overworld_netherite.descr": "感谢你下载Overworld Netherite! <3",
```

说明性文本一般不用谦称

在1.16或更高版本下，英文字母、阿拉伯数字等非中文文本无需在前后添加空格。
具体可见我们的 [翻译规范](https://cfpa.site/TransRules/) 中 **04.排版** 一节下。
诸如这类的问题请在您翻译的所有文件中全部查明并修改。